HiP-Achievements.Sdk/AchievementsService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace PaderbornUniversity.SILab.Hip.Achievements
+{
+    /// <summary>
+    /// A service that can be used with ASP.NET Core dependency injection.
+    /// Usage: In ConfigureServices():
+    /// <code>
+    /// services.Configure&lt;AchievementsConfig&gt;(Configuration.GetSection("Endpoints"));
+    /// services.AddSingleton&lt;AchievementsService&gt;();
+    /// </code>
+    /// </summary>
+    public class AchievementsService
+    {
+        private readonly AchievementsConfig _config;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public AchievementsService(IOptions<AchievementsConfig> config, ILogger<AchievementsService> logger,
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _config = config.Value;
+            _httpContextAccessor = httpContextAccessor;
+
+            if (string.IsNullOrWhiteSpace(config.Value.AchievementsHost))
+                logger.LogWarning($"{nameof(AchievementsConfig.AchievementsHost)} is not configured correctly!");
+        }
+
+        public ExhibitsVisitedClient ExhibitsVisited => new ExhibitsVisitedClient(_config.AchievementsHost)
```

Comment:
I think it's hard to see which of the properties refer to actions and which refer to achievements, maybe add a corresponding suffix? (e.g. "ExhibitsVisitedAchievement")